client/src/lib/solana/instruction.ts

```diff
@@ -41,16 +41,6 @@ export class SolInstruction extends Enum {
   }
 }
 
-export async function getKeyFromAuthority(
```

Comment:
OK, I did not realize that this code was in instruction.rs before. We needed move it out due to a cyclic dependency. Maybe the bigger question is why this needs to be called from `sol-data` after the refactor.